src/features/settings/components/RolesList.tsx

```diff
@@ -0,0 +1,120 @@
+import { FC } from 'react';
+import { GridColDef } from '@mui/x-data-grid-pro';
+import { Typography } from '@mui/material';
+
+import messageIds from '../l10n/messageIds';
+import RolesTable from './RolesTable';
+import { useMessages } from 'core/i18n';
+import useRoles from '../hooks/useRoles';
+import ZUIPersonAvatar from 'zui/ZUIPersonAvatar';
+import ZUIPersonHoverCard from 'zui/ZUIPersonHoverCard';
+
+type RolesListProps = {
+  orgId: number;
+};
+
+const RolesList: FC<RolesListProps> = ({ orgId }) => {
+  const messages = useMessages(messageIds);
+  const roles = useRoles(orgId);
```

Comment:
I think it would be better to have `useRoles(orgId).data || []` then variable 'roles' feels more right to it, and also can get rid of question marks in `roles?.data?`